src/lib/devConsole.js

```diff
@@ -1291,6 +1291,97 @@ const setLogForwarding = async (organizationCode, projectId, workspaceId, meshId
 	}
 };
 
+/**
+ * @param {string} organizationCode - The IMS org code
+ * @param {string} projectId - The project ID
+ * @param {string} workspaceId - The workspace ID
+ * @param {string} meshId - The mesh ID
+ */
+const getLogForwarding = async (organizationCode, projectId, workspaceId, meshId) => {
+	const { accessToken } = await getDevConsoleConfig();
+	const config = {
+		method: 'GET',
+		url: `${SMS_BASE_URL}/organizations/${organizationCode}/projects/${projectId}/workspaces/${workspaceId}/meshes/${meshId}/log/forwarding`,
+		headers: {
+			'Authorization': `Bearer ${accessToken}`,
+			'Content-Type': 'application/json',
+			'x-request-id': global.requestId,
+			'x-api-key': SMS_API_KEY,
+		},
+	};
+
+	logger.info(
+		'Initiating POST %s',
+		`${SMS_BASE_URL}/organizations/${organizationCode}/projects/${projectId}/workspaces/${workspaceId}/meshes/${meshId}/log/forwarding`,
+	);
+
+	try {
+		const response = await axios(config);
+
+		logger.info('Response from GET %s', response.status);
+
+		if (response?.status === 200) {
+			logger.info(`Get log forwarding configuration: ${objToString(response, ['data'])}`);
+			return {
+				data: response.data,
+			};
+		} else {
+			// not 200 response
+			logger.error(
+				`Something went wrong: ${objToString(
+					response,
+					['data'],
+					'Unable to get log forwarding details.',
+				)}. Received ${response.status}, expected 200`,
+			);
+			throw new Error(response.data.message);
+		}
+	} catch (error) {
+		if (error.response && error.response.status === 400) {
+			// The request was made and the server responded with a 400 status code
+			logger.error('Error getting log forwarding configuration: %j', error.response.data);
+
+			throw new Error('Invalid input parameters.');
+		} else if (error.response && error.response.status === 404) {
+			logger.error('Log forwarding details not found');
```

Comment:
Can we write a test to validate this message if customer has not setup log forwarding for a valid mesh